build/nuke/Build.Steps.cs

```diff
@@ -95,6 +95,20 @@ partial class Build
             );
         });
 
+    Target CompileManagedTests => _ => _
+        .Unlisted()
+        .Description("Compiles the managed code in the test directory")
+        .After(CompileManagedSrc)
```

Comment:
I think it's using these to order Targets. The code order seems not to matter and ignored.

Reply:
In this case, the `After` just means that if `CompileManagedSrc` was also specified as a build target, this would run afterwards. It's not like a `Depends` target, but if you wanted to enforce that dependency you could change to that